staging/src/k8s.io/legacy-cloud-providers/vsphere/nodemanager.go

```diff
@@ -21,14 +21,17 @@ package vsphere
 import (
 	"context"
 	"fmt"
+	"strconv"
 	"strings"
 	"sync"
 
 	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+	"k8s.io/klog/v2"
```

Comment:
nit: group this import with other k8s.io imports below

Reply:
done.